server/server.ts

```diff
@@ -2,7 +2,11 @@ import { config } from "dotenv";
 config();
 
 import fastify from "fastify";
+import cors from "@fastify/cors";
+import { userRoutes } from "./routes/users";
```

Comment:
lets have a space and try to use alias based imports

Reply:
I have tried alias based import for routes folder but for some reason it's throwing error and crashing the server.